<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCC::LanguageSettings</class>
 <widget class="QDialog" name="OCC::LanguageSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Language Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QComboBox" name="comboBox">
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_1">
     <property name="title">
      <string>Language settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Language</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Activate</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Default</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OCC::LanguageSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>361</x>
     <y>355</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OCC::LanguageSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>385</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
